### PR TITLE
Added documentation for the keba-charging-station component

### PR DESCRIPTION
**Pull request by dannerph**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24484

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9616"></a>

### CODE REVIEW

File: source/_components/keba.markdown [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keba.png
+ha_category:
+  - Binary Sensor
+  - Lock
+  - Sensor
+ha_release: 0.94
```
Comment (klaasnicolaas) [inline source/_components/keba.markdown]:
```suggestion
ha_release: 0.95
```

File: source/_components/keba.markdown [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/keba.markdown]: Please remove the `redirect_from`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/keba.markdown (hunk 2 above)
Comment (dannerph) [reply source/_components/keba.markdown]: Fixed, I probably used an old style documentation as template.

Comment (klaasnicolaas) [reply source/_components/keba.markdown]: Well what I mainly see is that people unquestionably take over without really knowing what it is for 😉

File: source/_components/keba.markdown [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      type: integer
+      default: 6
+    failsafe_save:
+      description: Saving the failsafe configuration to internal EEPROM of the Keba charging station. 1 means save it, 0 means do only keep this configuration until the next restart of the charging station (only if failsafe mode is enabled).
+      required: false
+      type: integer
+      default: 0
+{% endconfiguration %}
+
+## {% linkable_title Services %}
```
Comment (klaasnicolaas) [inline source/_components/keba.markdown]: Maybe put the service names in the titles

Comment (dannerph) [reply source/_components/keba.markdown]: I am not 100% sure what your suggestion is, but I now added the service names in the descritpion of the service and additionally added some example payloads to the services calls with parameters.

File: source/_components/keba.markdown [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keba.png
+ha_category:
+  - Binary Sensor
+  - Lock
+  - Sensor
+ha_release: 0.96
```
Comment (MartinHjelmare) [inline source/_components/keba.markdown]:
```suggestion
ha_release: 0.98
```

File: source/_components/keba.markdown [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+      default: 5
+{% endconfiguration %}
+
+## Services
+
+The `keba` component offers several services. Using these services will change the state of your charging station. So use these services with care!
+
+### Authorizing and Deauthorizing `keba.authorize` and `keba.deauthorize`
+
+The charging station can be authorize and deauthorize vai service calls (`keba.authorize` and `keba.deauthorize`) or via the lock component that is created automatically for the charging station. In both cases the RFID tag from the configuration is used.
```
Comment (MartinHjelmare) [inline source/_components/keba.markdown]:
```suggestion
The charging station can be authorized and deauthorized via service calls (`keba.authorize` and `keba.deauthorize`) or via the lock component that is created automatically for the charging station. In both cases the RFID tag from the configuration is used.
```

File: source/_components/keba.markdown [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+The service `keba.set_curr` sets the maximum current to the given current attribute in Ampere. Payload example:
+```json
+{
+  "current": 16.0
+}
+```
+
+### Request New Data `keba.request_data`
+
+The service `keba.request_data` send data update requests to the charging station.
```
Comment (MartinHjelmare) [inline source/_components/keba.markdown]:
```suggestion
The service `keba.request_data` sends data update requests to the charging station.
```

File: source/_components/keba.markdown [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+
+### Request New Data `keba.set_failsafe`
+
+The service `keba.set_failsafe` set the failsafe mode of the charging station. Payload example:
```
Comment (MartinHjelmare) [inline source/_components/keba.markdown]:
```suggestion
The service `keba.set_failsafe` sets the failsafe mode of the charging station. Payload example:
```

**Review by klaasnicolaas — DISMISSED** [review]
Think we can merge it now

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**